docs/kibana.md

```diff
@@ -0,0 +1,77 @@
+# Kibana and ARMOR setup
+
```

Comment:
In kibana 4.3.0 session store need to be enabled as final msearch request is not sending authorize header to backend even after successful authentication